### PR TITLE
Adds Pipfile support

### PR DESCRIPTION
**Pull request by tomdottom**
Addresses #18 and implements some of #10 

Current usage:
```
thanks package_name
thanks -r requirements.txt
thanks -p Pipefile
thanks package_name -r requirements.txt -r test_requirements.txt -p Pipfile

```

### CODE REVIEW

**Comment by tomdottom** [comment]
:thinking: failing tests. But `tox` passes locally :sob: 

Will have to return to this when I'm not so :sleeping:

File: thanks/cli.py [hunk 1]
```diff
@@ -1,20 +1,48 @@
...
-    return 0
+@click.argument("package_name",
+                nargs=-1)
+@click.option("--requirements", "-r",
+              multiple=True,
+              type=click.File("r"))
+@click.option("--pipfile", "-p",
+              multiple=True,
+              type=click.File("r"))
+# @click.option("--setuppy", "-s",
```
Comment (phildini) [inline thanks/cli.py]: I think it's good to support these in the future, but would rather not have commented out code lying around. If we want to add a `# TODO: Support more types` I'm fine with that.

File: thanks/thanks.py [hunk 2]
```diff
@@ -95,9 +87,51 @@ def get_remote_data(self, project_name):
...
+                    "the project repository asking the author to ",
+                    colored("ADD A 'FUNDING' PROJECT_URL ", "yellow"),
+                    "to the projects setup.py"
+                ]),
+                attrs=['bold']
+        ))
+
+        return '\n'.join(lines)
+
+    def rocks(self, colored_output=True):
```
Comment (phildini) [inline thanks/thanks.py]: could we instead call this `give` or `print`? I was confused reading this what `rocks` was referring to.

**Review by phildini — COMMENTED** [review]
Overall I think this is _amazing_, thank you. I have a couple small comments, but I think this is mostly ready-to-ship.

**Comment by tomdottom** [comment]
Ok @phildini, anything else?

**Review by phildini — APPROVED** [review]

**Pull request merged**